odoo/tools/config.py

```diff
@@ -335,6 +332,19 @@ def _build_cli(self):
                          help="Absolute path to the GeoIP Country database file.")
         parser.add_option_group(group)
 
+        group = optparse.OptionGroup(parser, "Shell options")
+        group.add_option(
+            '--shell-file', dest='shell_file',
+            help="Specify a python script to be run after the start of the shell. "
+                 "Overrides the env variable PYTHONSTARTUP.",
+            type="string", file_exportable=False)
+        group.add_option(
+            '--shell-interface', dest='shell_interface',
+            help="Specify a preferred REPL to use in shell mode. "
+                 "Supported REPLs are: [ipython|ptpython|bpython|python]",
+            type="string", file_exportable=False)
```

Comment:
We might use the opportunity to re-evaluate why we don't export the option upon `--save`.
```suggestion
            help="Specify a python script to be run after the start of the shell. "
                 "Overrides the env variable PYTHONSTARTUP.",
            type="string")
        group.add_option(
            '--shell-interface', dest='shell_interface',
            help="Specify a preferred REPL to use in shell mode. "
                 "Supported REPLs are: [ipython|ptpython|bpython|python]",
            type="string")
```